README.md

```diff
@@ -9,10 +9,18 @@ The LLVM-MOS compiler toolchain and platform libraries.
 - Atari 2600
   - 4K cartridge
   - [TigerVision 3E (2-KiB fixed, 2-KiB banked ROM or RAM)](https://www.taswegian.com/WoodgrainWizard/tiki-index.php?page=3E)
+- [Atari 5200](https://en.wikipedia.org/wiki/Atari_5200) Super Cart from [32 KiB](https://github.com/atari800/atari800/blob/master/DOC/cart.txt#L105)
```

Comment:
These should be GitHub permalinks; the line ordering may yet change. (The last update to `cart.txt` was 9 months ago.)

Reply:
Oh, they were supposed to be against a tagged release.  Does that suffice?  People don't usually change tags.

Reply:
Yeah, that should be fine.